<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Window3DSettings</class>
 <widget class="QGroupBox" name="Window3DSettings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>243</width>
    <height>165</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>GroupBox</string>
  </property>
  <property name="title">
   <string>Window settings</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QGridLayout" name="windowLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>x ∈</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>[</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QLineEdit" name="xMin">
       <property name="styleSheet">
        <string notr="true">width:4ch;
min-width:2em;</string>
       </property>
       <property name="inputMask">
        <string/>
       </property>
       <property name="text">
        <string>-5</string>
       </property>
      </widget>
     </item>
     <item row="0" column="3">
      <widget class="QLabel" name="label_7">
       <property name="text">
        <string>,</string>
       </property>
      </widget>
     </item>
     <item row="0" column="4">
      <widget class="QLineEdit" name="xMax">
       <property name="styleSheet">
        <string notr="true">width:4ch;
min-width:2em;</string>
       </property>
       <property name="text">
        <string>5</string>
       </property>
      </widget>
     </item>
     <item row="0" column="5">
      <widget class="QLabel" name="label_9">
       <property name="text">
        <string>]</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>y ∈</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>[</string>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QLineEdit" name="yMin">
       <property name="styleSheet">
        <string notr="true">width:4ch;
min-width:2em;</string>
       </property>
       <property name="text">
        <string>-5</string>
       </property>
      </widget>
     </item>
     <item row="1" column="3">
      <widget class="QLabel" name="label_8">
       <property name="text">
        <string>,</string>
       </property>
      </widget>
     </item>
     <item row="1" column="4">
      <widget class="QLineEdit" name="yMax">
       <property name="styleSheet">
        <string notr="true">width:4ch;
min-width:2em;</string>
       </property>
       <property name="text">
        <string>5</string>
       </property>
      </widget>
     </item>
     <item row="1" column="5">
      <widget class="QLabel" name="label_10">
       <property name="text">
        <string>]</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>z ∈</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLabel" name="label_11">
       <property name="text">
        <string>[</string>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QLineEdit" name="zMin">
       <property name="styleSheet">
        <string notr="true">width:4ch;
min-width:2em;</string>
       </property>
       <property name="text">
        <string>-5</string>
       </property>
      </widget>
     </item>
     <item row="2" column="3">
      <widget class="QLabel" name="label_12">
       <property name="text">
        <string>,</string>
       </property>
      </widget>
     </item>
     <item row="2" column="4">
      <widget class="QLineEdit" name="zMax">
       <property name="styleSheet">
        <string notr="true">width:4ch;
min-width:2em;</string>
       </property>
       <property name="text">
        <string>5</string>
       </property>
      </widget>
     </item>
     <item row="2" column="5">
      <widget class="QLabel" name="label_13">
       <property name="text">
        <string>]</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="lightLayout">
     <item>
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>Light source: (</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lightX">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="styleSheet">
        <string notr="true">width:4ch;
min-width:2em;</string>
       </property>
       <property name="text">
        <string>5</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_14">
       <property name="text">
        <string>,</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lightY">
       <property name="styleSheet">
        <string notr="true">width:4ch;
min-width:2em;</string>
       </property>
       <property name="text">
        <string>-5</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_15">
       <property name="text">
        <string>,</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lightZ">
       <property name="styleSheet">
        <string notr="true">width:4ch;
min-width:2em;</string>
       </property>
       <property name="text">
        <string>5</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_16">
       <property name="text">
        <string>)</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QCheckBox" name="showAxesBox">
     <property name="text">
      <string>Show axes</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>xMin</tabstop>
  <tabstop>xMax</tabstop>
  <tabstop>yMin</tabstop>
  <tabstop>yMax</tabstop>
  <tabstop>zMin</tabstop>
  <tabstop>zMax</tabstop>
  <tabstop>lightX</tabstop>
  <tabstop>lightY</tabstop>
  <tabstop>lightZ</tabstop>
  <tabstop>showAxesBox</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>xMin</sender>
   <signal>editingFinished()</signal>
   <receiver>Window3DSettings</receiver>
   <slot>onChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>128</x>
     <y>51</y>
    </hint>
    <hint type="destinationlabel">
     <x>165</x>
     <y>0</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>xMax</sender>
   <signal>editingFinished()</signal>
   <receiver>Window3DSettings</receiver>
   <slot>onChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>223</x>
     <y>40</y>
    </hint>
    <hint type="destinationlabel">
     <x>242</x>
     <y>13</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>yMin</sender>
   <signal>editingFinished()</signal>
   <receiver>Window3DSettings</receiver>
   <slot>onChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>110</x>
     <y>63</y>
    </hint>
    <hint type="destinationlabel">
     <x>86</x>
     <y>24</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>yMax</sender>
   <signal>editingFinished()</signal>
   <receiver>Window3DSettings</receiver>
   <slot>onChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>225</x>
     <y>77</y>
    </hint>
    <hint type="destinationlabel">
     <x>242</x>
     <y>28</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>zMin</sender>
   <signal>editingFinished()</signal>
   <receiver>Window3DSettings</receiver>
   <slot>onChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>73</x>
     <y>85</y>
    </hint>
    <hint type="destinationlabel">
     <x>39</x>
     <y>0</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>zMax</sender>
   <signal>editingFinished()</signal>
   <receiver>Window3DSettings</receiver>
   <slot>onChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>165</x>
     <y>91</y>
    </hint>
    <hint type="destinationlabel">
     <x>198</x>
     <y>11</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>lightX</sender>
   <signal>editingFinished()</signal>
   <receiver>Window3DSettings</receiver>
   <slot>onChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>113</x>
     <y>121</y>
    </hint>
    <hint type="destinationlabel">
     <x>57</x>
     <y>4</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>lightY</sender>
   <signal>editingFinished()</signal>
   <receiver>Window3DSettings</receiver>
   <slot>onChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>177</x>
     <y>116</y>
    </hint>
    <hint type="destinationlabel">
     <x>187</x>
     <y>18</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>lightZ</sender>
   <signal>editingFinished()</signal>
   <receiver>Window3DSettings</receiver>
   <slot>onChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>199</x>
     <y>127</y>
    </hint>
    <hint type="destinationlabel">
     <x>215</x>
     <y>26</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <signal>showAxes(bool)</signal>
  <slot>onChanged()</slot>
 </slots>
</ui>
